substrate/client/transaction-pool/src/fork_aware_txpool/tx_mem_pool.rs

```diff
@@ -539,20 +547,25 @@ where
 				Err(_) |
 				Ok(Err(TransactionValidityError::Unknown(_))) |
 				Ok(Err(TransactionValidityError::Invalid(_))) => {
-					log::trace!(
+					tracing::trace!(
 						target: LOG_TARGET,
-						"[{:?}]: Purging: invalid: {:?}",
-						xt_hash,
-						validation_result,
+						?xt_hash,
```

Comment:
```suggestion
						tx_hash = ?xt_hash,
```

... or rename `xt_hash` to `tx_hash` (we need same `tx_hash` field in all logs which refer to single transcation.